static/app/views/dashboards/widgets/tableWidget/tableWidgetVisualization.tsx

```diff
@@ -0,0 +1,86 @@
+import type {CSSProperties} from 'react';
+import {Fragment} from 'react';
+import {useTheme} from '@emotion/react';
+
+import type {GridColumnOrder} from 'sentry/components/gridEditable';
+import GridEditable from 'sentry/components/gridEditable';
+import type {TableData} from 'sentry/utils/discover/discoverQuery';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {
+  renderDefaultBodyCell,
+  renderDefaultHeadCell,
+} from 'sentry/views/dashboards/widgets/tableWidget/defaultTableCellRenderers';
+import type {TableColumn} from 'sentry/views/discover/table/types';
+
+interface TableWidgetVisualizationProps {
+  columns: Array<TableColumn<string>>;
+  loading: boolean;
+  tableData: TableData;
+  fitMaxContent?: boolean;
```

Comment:
I think I might've seen a comment from Jonas about changing this from a flag to a string literal so it can be scoped to different kinds of fit types, did we decide not to do that?